README.md

```diff
@@ -1 +1,49 @@
 # blocks-concurrent-batch-subscriber
+
+## Overview
+
+`blocks-concurrent-subscriber` subscribes the progresses of jobs with `blocks concurrent batch board` of [magellan-blocks](https://www.magellanic-clouds.com/blocks/).
+When the progresses are notified, `blocks-concurrent-subscriber` updates the status of the job and inserts the logs on mysql.
+
+`blocks-concurrent-subscriber` access to `blocks-concurrent-batch-agent` to get subscriptions to pull messages.
+
+## Install
+
+```bash
+$ go get github.com/groovenauts/blocks-concurrent-subscriber
+```
```

Comment:
@minimum2scp Thank you for your advice. But I mean it was an installation command not for developers but for users.
Then I realized that the "go get" installation is wrong for users by your comment, so I modified the description to suggest only downloading. 

(But we have no release files for the present, I'll make the files before the first release)